tests/integration/test_fit.py

```diff
@@ -109,17 +109,6 @@ def test_execute_model_nonmem(tmp_path, testdata):
         model = Model.parse_model('pheno_real.mod')
         model = model.replace(datainfo=model.datainfo.replace(path=tmp_path / 'pheno.dta'))
 
-        db = LocalDirectoryToolDatabase('db_model')
-
-        model = execute_model(model, db)
```

Comment:
Do we test `execute_model` with `db` elsewhere?

Reply:
Yes, within the same test just further down. The removed parts of the test were for a `Model` input object, which is no longer supported

Reply:
Ok, great!